docs/core/diagnostics/logging-tracing.md

```diff
@@ -1,110 +1,55 @@
-distributed across multiple machines or processes. This technique tracks requests through an
-application correlating together work done by different application components and separating
-it from other work the application may be doing for concurrent requests.
+### Trace
 
-## ILogger and logging frameworks
+<xref:System.Diagnostics.Trace?displayProperty=nameWithType> and <xref:System.Diagnostics.Debug?displayProperty=nameWithType> are .NET's oldest logging APIs. These have flexible configuration APIs and a large ecosystem of sinks, but only support unstructured logging. On .NET Framework they can be configured via an app.config file but in .NET Core there is no built-in file-based configuration mechanism. The .NET team continues to support these APIs for backward-compatability purposes but there is no plan to add new functionality. These are a fine choice for applications that are already using them. For newer apps that haven't already committed to a logging API, `ILogger` may offer better functionality.
 
-The low-level APIs may not be the right choice for your logging needs. You may want to consider a logging framework.
+## Specialized logging APIs
 
-The <xref:Microsoft.Extensions.Logging.ILogger> interface has been used to create a common logging interface where the loggers can be inserted through dependency injection.
+### Console
 
-For instance, to allow you to make the best choice for your application .NET offers support for a selection of built-in and third-party frameworks:
+The <xref:System.Console?displayProperty=nameWithType> class has the <xref:System.Console.Write%2A> and <xref:System.Console.WriteLine%2A> methods that can be used in simple logging scenarios. These APIs are very easy to get started with but the solution won't be as flexible as a general purpose logging API. Console only allows unstructured logging and there is no configuration support to select which log messages are enabled or to retarget to a different sink. Using the ILogger or Trace APIs with a console sink doesn't take much additional effort and keeps the logging configurable.
 
-- [.NET Built-in logging providers](../extensions/logging-providers.md#built-in-logging-providers)
-- [.NET Third-party logging providers](../extensions/logging-providers.md#third-party-logging-providers)
+### DiagnosticSource
 
-## Logging-related references
+<xref:System.Diagnostics.DiagnosticSource?displayProperty=nameWithType> is intended for logging where the log messages will be analyzed synchronously in-process rather than serialized to any storage. This allows the source and listener to exchange arbitrary .NET objects as messages whereas most logging APIs require the log event to be serializable. This technique can also be extremely fast, handling log events in 10s of nanoseconds if the listener is implemented efficiently. Tools that use these APIs often act more like in-process profilers though the API doesn't impose any constraint here.
```

Comment:
```suggestion
<xref:System.Diagnostics.DiagnosticSource?displayProperty=nameWithType> is intended for logging where the log messages will be analyzed synchronously in-process rather than serialized to any storage. This allows the source and listener to exchange arbitrary .NET objects as messages, whereas most logging APIs require the log event to be serializable. This technique can also be extremely fast, handling log events in tens of nanoseconds if the listener is implemented efficiently. Tools that use these APIs often act more like in-process profilers, though the API doesn't impose any constraint here.
```